provisioning/virtual-machines/ansible/roles/crownlabs/templates/cloud.cfg

```diff
@@ -4,35 +4,32 @@
 # A set of users which may be applied and/or used by various modules
 # when a 'default' entry is found it will reference the 'default_user'
 # from the distro configuration specified below
-# users:
-#   - default
+users:
+  - default
 
 # If this is set, 'root' will not be able to ssh in and they
 # will get a message to login instead as the default $user
 disable_root: true
 
 # This will cause the set+update hostname module to not operate (if true)
-preserve_hostname: true
+preserve_hostname: false
 
-# Disable networking configuration
-network:
-    config: disabled
 
 # The modules that run in the 'init' stage
 cloud_init_modules:
-# - migrator
+ - migrator
```

Comment:
Would it be possible to add a small comment for each line, telling why are the above modules needed?